source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -240,18 +240,36 @@ function New-StigCheckList
     $writer = [System.Xml.XmlWriter]::Create($OutputPath.FullName, $xmlWriterSettings)
 
     $writer.WriteStartElement('CHECKLIST')
-
-    #region ASSET
-
     $writer.WriteStartElement("ASSET")
 
+    try
```

Comment:
The contents of this try/catch should be a function that is invoked based on user need and therefore should require a switch parameter indicating those fields should be populated.  The idea would be to include all the logic in a single script block that's used with invoke-command.  Also, -ErrorAction stop may be required in order for the try/catch to work correctly.

Reply:
Added the switch parameter to include localhost data.
Consolidated logic into single invoke-command scriptblock.
Added '-ErrorAction Stop' switch.